smartsim/_core/control/controller.py

```diff
@@ -638,26 +636,26 @@ def _set_dbobjects(self, manifest: Manifest) -> None:
 
         for model in manifest.models:
             if not model.colocated:
-                for db_model in model._db_models:
+                for db_model in model.db_models:
```

Comment:
This change is exposing `db_models` to users where before it was protected. I'll defer to the rest of the team on the advisability of so doing

Reply:
It seems clear we do need them to be publicly accessible given that we're already touching the properties. Making the variable names match their accessibility is purely cosmetic.

A refactor that pushes functionality like this method from the controller into the ensemble (or model) could also solve this, but feels out of scope of this task.